### ISSUE TITLE
Uncaught TypeError: Cannot read property 'getPath' of null

### ISSUE THREAD

**Issue by Jackeriss** [AtomLinter/atom-minimap-linter#19]
1. update atom-minimap-linter
2. click restart when update is complete
3. bang!

**Atom**: 1.18.0 x64
**Electron**: 1.3.15
**OS**: Mac OS X 10.12.5
**Thrown From**: [minimap-linter](https://github.com/AtomLinter/atom-minimap-linter) package 2.0.1

### Stack Trace

Uncaught TypeError: Cannot read property 'getPath' of null

```
At /Users/jackeriss/.atom/packages/minimap-linter/lib/minimap-linter.js:119

TypeError: Cannot read property 'getPath' of null
    at renderOldMessages (/packages/minimap-linter/lib/minimap-linter.js:119:49)
```

### Commands

```
     -0:23.4.0 tree-view:show (atom-workspace.workspace.scrollbars-visible-when-scrolling)
```

### Non-Core Packages

```
atom-beautify 0.30.3 
atom-material-syntax 1.0.6 
atom-material-ui 2.0.4 
autoclose-html 0.23.0 
autocomplete-python 1.9.1 
busy-signal 1.4.3 
docker 0.7.0 
file-icons 2.1.8 
git-plus 7.9.3 
git-time-machine 1.5.9 
hyperclick 0.1.2 
intentions 1.1.2 
language-docker 1.1.8 
linter 2.2.0 
linter-docker 0.2.0 
linter-pylint 2.1.0 
linter-ui-default 1.6.2 
minimap 4.28.3 
minimap-linter 2.0.1 
minimap-pigments 0.2.2 
package-sync 1.1.0 
pigments 0.39.1 
platformio-ide-terminal 2.5.3 
python-indent 1.1.0 
python-tools 0.6.9 
```

**Comment by Arcanemagus** [comment]
Interesting, this should only be possible if you have lint messages for unsaved files... which Linter doesn't run on 😕.

In any case, I'll add a guard for that, thanks for reporting!

**Comment by Arcanemagus** [comment]
Published in v2.0.2. 🎉
